src/main/java/teammates/storage/api/OfyHelper.java

```diff
@@ -41,6 +43,19 @@ public static void registerEntityClasses() {
     @Override
     public void contextInitialized(ServletContextEvent event) {
         // Invoked by GAE at application startup.
+
+        // PRODUCTION
+        //ObjectifyService.init();
+
+        // TEST
+        ObjectifyService.init(new ObjectifyFactory(
+                DatastoreOptions.newBuilder()
+                        .setHost("http://localhost:8484")
+                        .setProjectId("tm-obj-v6-test")
+                        .build()
+                        .getService()
```

Comment:
For the host url sometimes `port 8484` is not available and gives an exception `Exiting due to exception: java.io.IOException: Failed to bind` (as discussed [here](https://github.com/Derek-Hardy/teammates/discussions/7)). It occurs after an initial termination of the emulator. I'm wondering if changing port number is the only solution, if it is then will it be cumbersome as we need to change the host url here everytime we use a different port number.

Reply:
(let's keep the discussion within main repo)

**Original troubleshoot discussion:**

> Sometimes the recommended emulator setup in [wiki](https://github.com/objectify/objectify/wiki/Setup#initialising-the-objectifyservice-to-work-with-emulator-applies-to-v6) might not work with `Exiting due to exception: java.io.IOException: Failed to bind`.
> 
> In that case, just try out some other localhost ports such as `gcloud beta emulators datastore start --host-port=localhost:8482`

It is a valid point. Currently I don't have a better proposal to fully resolve this. 🤔

Reply:
If we follow through with this solution where the port number is variable, it might be a good idea to set it in a config file (maybe `gradle.properties`?) so developers won't accidentally add the port number change to git.

Reply:
@tiuweehan Yup, have moved it to config file.